src/state_manager/mod.rs

```diff
@@ -582,7 +589,7 @@ where
             }
         };
         let result = self
-            .compute_tipset_state(Arc::clone(ts), Some(callback), VMTrace::NotTraced)
+            .compute_tipset_state(Arc::clone(ts), Some(callback), VMTrace::Traced)
```

Comment:
Do we want to keep `VMTrace::Traced` here?

Reply:
`trace` is part of the response, see line 581
```rust
execution_trace:
```